src/main/java/com/zufar/onlinestore/payment/endpoint/PaymentEndpoint.java

```diff
@@ -4,8 +4,8 @@
 import com.stripe.exception.StripeException;
 import com.zufar.onlinestore.payment.api.PaymentApi;
 import com.zufar.onlinestore.payment.dto.*;
```

Comment:
@Reyzis2021 zufar is talking about getting rid of wildcard that is comprehensive

Reply:
done